docker-compose/generate-config.py

```diff
@@ -19,6 +19,7 @@
     "master-docker-nonstandard",
```

Comment:
Probably rename it `master-docker-nonstandard-1` to be homogeneous.

Reply:
I was thinking to do this, but then I stopped when I realized that we need to change the service files. We really need to move to the docker pipeline for prod as well. I'll rename it though